content/developers/developer-patterns/scitt-api/index.md

```diff
@@ -0,0 +1,211 @@
+
+  cd datatrails-scitt-samples
+  ```
+
+## Environment Configuration
+
+1. Create a Python Virtual Environment for the sample scripts and install the dependencies
+
+    ```shell
+    python -m  venv venv
+    source venv/bin/activate && \
+    pip install -r requirements.txt
+    ```
+
+1. To ease copying and pasting commands, update any variables to fit your environment
+
+  ```shell
+  ISSUER=sample.sysnation.dev
+  SIGNING_KEY=my-signing-key.pem
+  ```
+
+1. Create a [bearer_token](/developers/developer-patterns/getting-access-tokens-using-app-registrations) stored as a file, in a secure local directory with 0600 permissions.
+
+## Create a Signing Key
+
+{{< note >}}
+If you already have a COSE Key, skip ahead to [Register a SBOM for the Artifact](#register-a-sbom-for-the-artifact)
+{{< /note >}}
+
+DataTrails supports multiple signing keys.
+_(TODO: link to supported DataTrails Signing Keys\)_<br>
+For the Quickstart, create a testing [COSE Key](https://cose-wg.github.io/cose-spec/#key-structure) which DataTrails will cryptographically validate upon registration
+
+  ```shell
+  openssl ecparam -name prime256v1 -genkey -out $SIGNING_KEY
+  ```
+
+## Register a SBOM for the Artifact
+
+When registering statements about an artifact, a common identifier (Feed) is required to correlate a collection of statements.
+
+The example generates an [SPDX SBOM](https://spdx.dev/), which generates a [Document Namespace](https://spdx.github.io/spdx-spec/v2.2.2/document-creation-information/#65-spdx-document-namespace-field) field which is used for the `CWT_Claims subject` property.
+
+_\<TODO: Add a doc for creating identifiers>_
+
+1. Using the [SBOM Tool](https://github.com/microsoft/sbom-tool), generate an SPDX SBOM for `artifact.js`
+
+    ```bash
+    sbom-tool generate -D true \
+      -b scitt/artifacts \
+      -ps scitt-community \
+      -pn scitt-nodejs-example -pv 0.0.0 \
+      -nsb https://scitt.io/examples
+    ```
+
+    [More info for the SPDX sbom-tool](https://github.com/microsoft/sbom-tool/blob/main/docs/sbom-tool-arguments.md)
+1. Capture the Document Namespace for the SCITT `subject`
+
+    ```shell
+    FEED=$(cat scitt/artifact/_manifest/spdx_2.2/manifest.spdx.json | jq -r .documentNamespace)
+    ```
+
+1. Create a Signed Statement for the SPDX SBOM
+
+    ```shell
+    python scitt/create_signed_statement.py \
+      --signing-key-file $SIGNING_KEY \
+      --issuer $ISSUER \
+      --feed $FEED \
+      --content-type application/spdx+json \
+      --payload scitt/artifacts/_manifest/spdx_2.2/manifest.spdx.json \
+      --output-file signed-statement-sbom.cbor
+
+1. Register the SBOM
+
+    ```shell
+
+    ENTRY_ID=$(statement-register.py \
+      issuer $ISSUER \
+      signing-key $SIGNING_KEY \
+      feed $FEED \
+      payload artifacts/_manifest/spdx_2.2/manifest.spdx.json \
+      content-type application/spdx+json)
+    ```
+
+1. Retrieve a SCITT Receipt
+
+    ```shell
+    statement-get-receipt.py $ENTRY_ID
+    ```
```

Comment:
We get the operationID from the response of the register request.

Again please put in auth for the request.
```suggestion
    ```shell
    curl https://app.datatrails.ai/archivist/v1/publicscitt/operations/$OPERATION_ID
    ```
```